code/modules/hydroponics/goon/weed_seeds.dm

```diff
@@ -0,0 +1,30 @@
+/datum/seed/slurrypod
+	name = "slurrypod"
+	seed_name = "slurrypod"
+	display_name = "slurrypod"
+	mutants = list("omegaslurrypod")
+	packet_icon = "seed-slurrypod"
+	plant_icon = "slurrypod"
+	chems = list("toxin" = list(1,10))
```

Comment:
You can actually use Toxic Slurry for this and the chem_splash var now---additionally you can use glowing slurry in place of "omega_mutagen".